tests/integration/components/tree-test.js

```diff
@@ -0,0 +1,90 @@
+import { render, fillIn } from "@ember/test-helpers";
+import { hbs } from "ember-cli-htmlbars";
+import { setupMirage } from "ember-cli-mirage/test-support";
+import { setupIntl } from "ember-intl/test-support";
+import { setupRenderingTest } from "ember-qunit";
+import { module, test } from "qunit";
+
+module("Integration | Component | tree", function (hooks) {
+  setupRenderingTest(hooks);
+  setupMirage(hooks);
+  setupIntl(hooks, ["de"]);
+  hooks.beforeEach(async function () {
+    const root = this.server.create("scope");
+    const level1 = this.server.createList("scope", 3, {
+      level: 1,
+      parent: root,
+    });
+    const grandchildren = this.server.createList("scope", 2, {
+      level: 2,
+      parent: level1[0],
+    });
+
+    grandchildren.forEach((grandchild) => {
+      grandchild.findParents = () => {
```

Comment:
Could we lift this somehow into the factory?

Reply:
Fixed :upside_down_face: